src/components/SignUp.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+  <section class="section section bg-soft pb-5 overflow-hidden z-2">
+            <div class="container z-2">
+            <SplashScreen></SplashScreen>
```

Comment:
Why is the `<SplashScreen/>` used in signup component ?

Reply:
@itisdb please check this.

Reply:
The splash screen component had image in it as well. That's the reason!

Reply:
Oh LOL ! Can we use the image directly here ?

Reply:
![image](https://user-images.githubusercontent.com/53273486/121817258-de832300-cc9d-11eb-8ac4-7ed4f72ef01b.png)
![image](https://user-images.githubusercontent.com/53273486/121817269-e93db800-cc9d-11eb-9ab4-49a8fdbacc62.png)


The Code has been updated.